src/lib/config.js

```diff
@@ -30,7 +30,7 @@ class Config {
     this.timezone = 0;
 
     this.locale = 'enUS';
-    this.os = 'Mac';
+    this.os = 'Win';
```

Comment:
As i said you TC does multiple checks in core and if platform is not Win or OSX it doesn't allow you connecting to worldserver.

However if you don't believe me i've found the line for you: https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/game/Server/WorldSocket.cpp#L471